examples/replay/replay_mover.py

```diff
@@ -162,13 +162,42 @@ def run(self, job_id):
             localtime.stop()
 
         walltime.stop("Total Walltime")
+    
+    def regrid_to_gaussian(
+            self,
+            xds: xr.Dataset,
+            gaussian_grid_path: str,
+    ) -> xr.Dataset:
+        """Regrid (if needed) - will be used of ocean/sea ice data"""
+        gaussian_grid = xr.open_zarr(
+            gaussian_grid_path,
+            storage_options={"token": "anon"},
+            )
+        lons = gaussian_grid['grid_xt'].values
+        lats = gaussian_grid['grid_yt'].values
+        rg = MOM6Regridder(
```

Comment:
```suggestion
        rg = self.Regridder(
```

Reply:
And could you modify the __init__ method with something like this:

```python
        self.Regridder = None
        if component.lower() == "fv3":
            self.Dataset = FV3Dataset
            self.cached_path = self.fv3_path
            if "regrid" in self.config.keys():
                raise NotImplementedError
        elif component.lower() == "mom6":
            self.Dataset = MOM6Dataset
            self.cached_path = self.mom6_path
            self.Regridder = MOM6Regridder
        elif component.lower() == "cice6":
            self.Dataset = CICE6Dataset
            self.cached_path = self.cice6_path
            self.Regridder = CICE6Regridder

Reply:
done.